site/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "build": "gatsby clean && gatsby build",
-    "develop": "gatsby clean && gatsby develop",
+    "develop": "gatsby clean && GATSBY_SITE_URL=http://localhost:8000 gatsby develop",
```

Comment:
`GATSBY_SITE_URL=http://localhost:8000` why?

Reply:
this is to produce correct URLs for icons when run in development, we need to know the full (public) URL to produce the urls for images correctly (as twitter doesn't support site relative), so i've used it everywhere we generate these URLs.

Reply:
Ok, just asking because I run the app on `8001` because `8000` is already in use by default for my main project 😄 

Thanks for explaining 👍